tla-types/src/main/scala/at/forsyte/apalache/tla/types/package.scala

```diff
@@ -1,22 +1,134 @@
 package at.forsyte.apalache.tla
 
+/**
+ * Defines a premade instancce of [[at.forsyte.apalache.tla.typecomp.ScopedBuilder ScopedBuilder]], named [[tla]].
+ *
+ * =Importing and using [[tla]]=
+ *
```

Comment:
This jumps right into the nitty-gritty of how to convert a `TBuilderInternalState` to `TlaEx`, but the reader hasn't seen how to even build an expression.
I would include a brief explanation on how to build expressions first, with an example.

```suggestion
 *
 * The [[tla]] object provides a convenient interface for constructing type- and scope-correct TLA+ expressions.
 * It exposes methods that correspond to TLA+ operators, for example:
 * {{{
 * val ex: TlaEx = tla.plus(tla.int(2), tla.int(3))
 * }}}
 * 
```